docs/tutorials/aws.md

```diff
@@ -971,6 +971,8 @@ Route53 has a [5 API requests per second per account hard quota](https://docs.aw
 Running several fast polling ExternalDNS instances in a given account can easily hit that limit. Some ways to reduce the request rate include:
 * Reduce the polling loop's synchronization interval at the possible cost of slower change propagation (but see `--events` below to reduce the impact).
   * `--interval=5m` (default `1m`)
+* Enable a Cache to store the zone records list. It comes with a cost: slower propagation when the zone gets modified from other sources.
```

Comment:
Can you explain this part a bit better? What do you mean with "other sources"?

Reply:
Sure, here you go

```suggestion
* Enable a Cache to store the zone records list. It comes with a cost: slower propagation when the zone gets modified from other sources such as the AWS console, terraform, cloudformation or anything similar.
```

Reply:
proposed changes sounds good to me